.buildkite/init-settlements.yml

```
@@ -215,16 +226,20 @@ steps:
       done
     depends_on: "notification-setup-init"
     allow_dependency_failure: true
-    if: "build.env('NOTIFY_PSR_FEED') != null"
+    if: "build.env('NOTIFY_DISCORD_FEED') != null"
 
   - wait: ~
     continue_on_failure: true
 
-  - label: ":gear: Upload and trigger fund-settlements pipeline"
+  - label: ":gear: Upload and get running fund-settlements pipeline"
     commands:
       - epoch=$(buildkite-agent meta-data get epoch)
       - claim_type=$(buildkite-agent meta-data get claim_type)
-      - 'echo "Epoch: $$epoch, Claim type: $$claim_type"'
+      - slack_feed=$(buildkite-agent meta-data get slack_feed)
+      - export EPOCH="$$epoch"
+      - export CLAIM_TYPE="$$claim_type"
+      - export SLACK_FEED="$$slack_feed"
+      - 'echo "Epoch: $$epoch, Claim type: $$claim_type, Slack Feed: $$slack_feed"'
```

```suggestion
      - echo "Epoch: '$$epoch', Claim Type: '$$claim_type', Slack Feed: '$$slack_feed'"
```
(non-blocking - minor inconsistency)

---

A note on this above.